test/common/common/log_macros_test.cc

```diff
@@ -37,4 +37,48 @@ TEST(Logger, All) {
   // Misc logging with no facility.
   ENVOY_LOG_MISC(info, "fake message");
 }
+
+TEST(Logger, evaluateParams) {
+  uint32_t i = 1;
+
+  // set logger's level to low level.
+  // log message with higher severity and make sure that params were evaluated.
+  GET_MISC_LOGGER().set_level(spdlog::level::info);
+  ENVOY_LOG_MISC(warn, "test message '{}'", i++);
+  ASSERT_THAT(i, testing::Eq(2));
+}
+
+TEST(Logger, doNotEvaluateParams) {
+  uint32_t i = 1;
+
+  // set logger's logging level high and log a message with lower severity
```

Comment:
Super nitty, mind set ->Set here and above and do -> Do on 69?

Reply:
Thanks for catching it. It has been corrected.